client/my-sites/current-site/index.jsx

```diff
@@ -151,7 +152,9 @@ module.exports = React.createClass( {
 						homeLink={ true }
 						enableActions={ true }
 						externalLink={ true }
-						onSelect={ this.trackHomepageClick }
+						onClick={ this.previewSite }
+						onSelect={ this.previewSite }
```

Comment:
AFAICT `onSelect` translates into the `onTouchTap` event? I'd assume we want that.